_community_videos/01_talk_series.md

```diff
@@ -0,0 +1,21 @@
+---
+title: PyWhy Causality in Practice | LLMs for causal inference
+slug: pywhy-video
+layout: page
+description: >-
+  PyWhy community Discord and meeting schedule. 
+summary: >-
+
+  Emre Kiciman, Senior Principal Researcher at Microsoft, talks about pywhy-llm, a new experimental library that focuses on using large language models for causality.
+  <br>
+  <br>
+  PyWhy Causality in Practice is a new talk series focusing on causality and machine learning, especially from a practical perspective. We'll have tutorials and presentations about PyWhy libraries but also talks by external speakers working on causal inference.
```

Comment:
this sentence can be moved up to a global section.

Reply:
> this sentence can be moved up to a global section.

Let me know if this looks good to you. If so, I can add a commit with this change. 
<img width="534" alt="image" src="https://github.com/py-why/py-why.github.io/assets/22120253/80eb7dc5-984b-475e-b95b-915f60fb5615">

Reply:
yes, this looks good!

Reply:
Done